installing_on_kubernetes/_topics/cluster_image_policy.md

```diff
@@ -0,0 +1,26 @@
+## Allow docker.io/manageiq images in kubernetes
+
+Depending on your cluster's configuration, kubernetes may not allow deployment of images from `docker.io/manageiq`.  If so, deploying the operator may raise an error:
+```
+Error from server: error when creating "deploy/operator.yaml": admission webhook "trust.hooks.securityenforcement.admission.xxx" denied the request:
+Deny "docker.io/manageiq/manageiq-operator:latest", no matching repositories in ClusterImagePolicy and no ImagePolicies in the "YYY" namespace
+```
+To allow images from `docker.io/manageiq`, edit the clusterimagepolicies and add `docker.io/manageiq/*` to the list of allowed repositories:
```

Comment:
I can't suggest here, but for consistency leave a blank line above and below the triple backticked block.